code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -18,35 +18,43 @@
 	pressure_resistance = 7 * ONE_ATMOSPHERE
 	req_access = list()
 
+	///Is the valve open?
 	var/valve_open = FALSE
-	var/release_log = ""
```

Comment:
why are you killing logging here?

Reply:
this var is unused that's why i removed it, if you check the code you'll see that the only other instance of this var being used is `release_log += logmsg` in the UI proc, but unless i'm wrong and this var get's used somewhere else in some other way, this doesn't log a thing

Reply:
unless is for admins to be able to check the canister with VV

Reply:
Might want to test on local and find out.

Reply:
yeah it was to log on VV, i restored it, thanks for bringing it up!